includes/class-author-update-watcher.php

```diff
@@ -0,0 +1,170 @@
+<?php
+/**
+ * Newspack Network watcher for updates on authors' information
+ *
+ * @package Newspack
+ */
+
+namespace Newspack_Network;
+
+/**
+ * Class to handle the plugin admin pages
+ */
+class Author_Update_Watcher {
+
+	/**
+	 * Flag to indicate if the processing of the author updated event is in progress.
+	 * If true, this class won't fire any events to avoid an infinite loop in author updates.
+	 *
+	 * @var boolean
+	 */
+	public static $processing_author_updated_event = false;
+
+	/**
+	 * Holds information about the authors that were updated in this request, if any.
+	 *
+	 * @var array
+	 */
+	public static $updated_authors = [];
+
+	/**
+	 * The metadata we're watching and syncing.
+	 *
+	 * @var array
+	 */
+	public static $watched_meta = [
+		// Social Links.
+		'facebook',
+		'instagram',
+		'linkedin',
+		'myspace',
+		'pinterest',
+		'soundcloud',
+		'tumblr',
+		'twitter',
+		'youtube',
+		'wikipedia',
+
+		// Core bio.
+		'description',
+
+		// Newspack.
+		'newspack_job_title',
+		'newspack_role',
+		'newspack_employer',
+		'newspack_phone_number',
+
+		// Yoast SEO.
+		'wpseo_title',
+		'wpseo_metadesc',
+		'wpseo_noindex_author',
+		'wpseo_content_analysis_disable',
+		'wpseo_keyword_analysis_disable',
+		'wpseo_inclusive_language_analysis_disable',
+	];
+
+	/**
+	 * The user properties we're watching and syncing.
+	 *
+	 * @var array
+	 */
+	public static $user_props = [
+		'display_name',
```

Comment:
Shouldn't first and last name also be synchronised?

Reply:
We talked about only syncing what's displayed on the front-end of the site, so display name is enough. But it's a valid question

Reply:
> To pull data, I've ran wp cron event run newspack_network_pull_from_hub

Nice, you can also do it via the UI now, there's a button there "Sync now".

And in case you've missed it, there are detailed docs about how to follow the events travelling between sites:

https://github.com/Automattic/newspack-network/blob/master/DEV_NOTES.md#troubleshooting

Reply:
> We talked about only syncing what's displayed on the front-end of the site

What prompted me to ask is that name is displayed on the user list on the admin panel. For the administrator of a hub site, it'd be nice to see the full name instead of a sad "–":

<img width="562" alt="image" src="https://github.com/Automattic/newspack-network/assets/7383192/14812d52-9d68-4634-9fbc-05d9fc09c5ed">

Reply:
ah, good call!

Reply:
DOne